packages/now-client/src/create-deployment.ts

```diff
@@ -53,6 +53,20 @@ export default function buildCreateDeployment(
 
     let rootFiles: string[];
 
+    if (Array.isArray(path)) {
+      for (const filePath of path) {
+        if (!isAbsolute(filePath)) {
+          yield { type: 'error', message: 'Provided path is not absolute' };
```

Comment:
```suggestion
          yield { type: 'error', message: `Provided path ${filePath} is not absolute` };
```

Perhaps this should say which path is not absolute?